internal/brokers/localbroker.go

```diff
@@ -5,21 +5,30 @@ import (
 	"errors"
 )
 
+//nolint:unused // Those should never be called
```

Comment:
Maybe with a better description why we need them, right? Like, we still need to associate the object, but with no implementation.